### PR TITLE
Feature/hiding backend menu points

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: src/Menu/AdminMenuListener.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        $userRoles = array_flip($this->user->getRoles());
+        $menusToHide = array_intersect_key($this->hiddenMenusConfig, $userRoles);
+        if (true === empty($menusToHide)) {
+            return;
+        }
+
+        $this->hideMenuEntries($menu, $menusToHide);
+    }
+
+    private function hideMenuEntries(ItemInterface $menu, $menusToHide)
```
Comment (michz) [inline src/Menu/AdminMenuListener.php]: no return type here?

Comment (michz) [reply src/Menu/AdminMenuListener.php]: no typehint for $menusToHide ? no annotation for $menusToHide?

File: tests/Behat/Context/Setup/AdminContext.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+
+class AdminContext implements Context
+{
+    /**
+     * @var SharedStorageInterface
```
Comment (michz) [inline tests/Behat/Context/Setup/AdminContext.php]: In behat Contexts we use other codestyle?

**Review by michz — CHANGES_REQUESTED** [review]

**Review by michz — DISMISSED** [review]

File: features/admin/browsing_admin_area.feature [hunk 3]
```diff
@@ -6,11 +6,17 @@ Feature: Browsing admin area
 
   Background:
     Given the store operates on a single channel in "United States"
-    And I am logged in as an administrator
+
+    # the config under tests/Application/app/config/config.yml configures the plugin
```
Comment (mattagohni) [inline features/admin/browsing_admin_area.feature]: is it intentional, that these steps are commented out?

Comment (sd-lueckel) [reply features/admin/browsing_admin_area.feature]: It is only a comment, that the config is important. It is not a step

Comment (sd-lueckel) [reply features/admin/browsing_admin_area.feature]: I added a additional comment, perhaps now it is more clear

**Review by mattagohni — APPROVED** [review]

**Review by michz — APPROVED** [review]

**Pull request merged**